src/EditorFeatures/CSharpTest/Classification/SyntacticClassifierTests.cs

```diff
@@ -1772,6 +1774,68 @@ class Bar { }";
                 Punctuation.CloseCurly);
         }
 
+        [Theory]
+        [CombinatorialData]
+        public async Task XmlDocComment_Langword_NormalKeywords(TestHost testHost)
+        {
+            var keywords = SyntaxFacts.GetKeywordKinds().Select(SyntaxFacts.GetText);
```

Comment:
I definitely want to extract keyword as a test parameter, but in such case I would need to use `CombinatorialMemberData`, which is not available because the version of `Xunit.Combinatorial` is too low. I can create a separate issue for it, or fix it here if you explain to me how to bump `Xunit.Combinatorial` version